src/main/java/com/lolduo/duo/repository/LoLUserRepository.java

```diff
@@ -7,6 +7,9 @@
 import java.util.List;
 
 public interface LoLUserRepository extends JpaRepository<LoLUserEntity, String> {
-    @Query(value = "select LoLUserEntity.puuid from LoLUserEntity where LoLUserEntity.tier = ?1")
+    //@Query(value = "select LoLUserEntity.puuid from LoLUserEntity where LoLUserEntity.tier = ?1")
+    //List<String> findPuuidsByLeague (String league);
+
+    @Query(value = "select puuid from lol_user where tier = ?1",nativeQuery = true)
```

Comment:
이걸 nativequery로 한 이유가 있는가?
"select LE.puuid from LoLUserEntity LE where LE.tier = ?1"

Reply:
왜 그런지 모르겠는데, 실행이 안댐.  관련 에러가 3개 정도뜨는데, 대충 쿼리를 만들 수 없다는 에러인걸로 해석되었음.
관련에서 한 15분정도 찾아봤는데, 여기서 시간 잡아먹히는건 아닌거 같아서 내가 잘 아는 nativequery로 처리하고 넘겼음.

Could not create query for public abstract java.util.List com.lolduo.duo.repository.LoLUserRepository.findPuuidsByLeague(java.lang.String);

Validation failed for query for method public abstract java.util.List com.lolduo.duo.repository.LoLUserRepository.findPuuidsByLeague(java.lang.String)!

java.lang.NullPointerException: Cannot read field "value" because "s1" is null

Reply:
위에 제가 적어준대로 한번 진행 해보시길

Reply:
오 별칭이 있으면 되는구나 희한하네. 되는 거 확인 후 수정commit 올림